src/components/select/tests/SingleSelectWithFilter.spec.tsx

```diff
@@ -179,7 +179,7 @@ describe('Select', () => {
                     .simulate('keydown', {keyCode: keyCode.enter})
                     .simulate('keyup', {keyCode: keyCode.enter});
 
-                expect(dispatchSpy).toHaveBeenCalledWith(selectListBoxOption(id, undefined, items[0].value));
+                expect(dispatchSpy).toHaveBeenCalledWith(selectListBoxOption(id, undefined, items[1].value));

```

Comment:
```
const expectedActiveItem = _.findWhere(items, {selected: true});
expect(dispatchSpy).toHaveBeenCalledWith(selectListBoxOption(id, undefined, expectedActiveElement);
```

would be more resilient, if someone refactor items for any reason, there is less chance that this test will break with this approach ⬆️